tests/queries/test_slippage_investigation.py

```diff
@@ -66,6 +66,40 @@ def test_no_outrageous_slippage(self):
         for obj in top_five_negative + top_five_positive:
             assert abs(int(obj["eth_slippage_wei"])) < 1 * 10**18
 
+    def test_positive_slippage_evaluation(self):
+        """
+        Transaction 0x5c4e410ce5d741f60e06a8621c6f12839ad39273f5abf78d4bbc1cd3b31de46c
```

Comment:
This is a very tough settlement to evaluate (Quasilabs went crazy and internalized 7 interactions, so I am pretty sure the Dune query miscalculated slippage here), but I guess the point of the test is to check that it returns the amount that is specified in the Dune query, and not to check the logic behind it.

Reply:
Well it would also be nice to include a test of something we can mentally evaluate as well. If you have another transaction in mind I would gladly add it. Otherwise I can try to dig one up tomorrow and add a second test of the same type.

Reply:
`0x2509839cda8a4fec0c3b0ae4e90537aef5afaaeeee88427161d1b08367aeb0d7`

This one is just one Univ3 interaction that resulted in 82 USDC positive slippage.

Reply:
Thanks will add.

Reply:
Just a note that the USD valuation we recovered on this transaction was 83.37 USD (note that USDC is usually a worth a bit more than 1 USDC -- like 1.01 or something).

Here is a cool query for [{{Token}} Price in {{Time}}Ball of Minute {{Radius}} for {{Blockchain}}](https://dune.com/queries/1853389)

Reply:
New test has been added with 83.37 as the test USD value.